### DISCUSSION TITLE
Documentation about layout?

### DISCUSSION THREAD

**Discussion by mmontone** [tabler/tabler#1917] [Q&A]
Hi. I cannot find documentation about layout, grid system, etc. Like, I want to work with columns. I could figure it out from the source of some of the examples. But is there a section with explanations in the docs? Thanks.

**Comment by ethancrawford** [comment]
@mmontone - in case you are still wondering about this, then no, not really. The closest you might get to an explanation in the Tabler docs is on the [Page layouts](https://docs.tabler.io/ui/layout/page-layouts) page. Tabler doesn't appear to be concerned with providing documentation for low-level things such as containers/grids/rows and columns etc. At this stage, you probably need to consult the [Bootstrap documentation](https://getbootstrap.com/docs/).
